tasks/br/data.py

```diff
@@ -240,8 +240,7 @@ def columns(self):
                     continue
 
                 col_id = line[0]
-                col_name_pt = line[1].decode('latin1')
-                col_name_en = line[2]
+                col_name_pt = line[1]
```

Comment:
Why do you remove the col_name_en column?

Reply:
flake8 warning about unused variable.

Reply:
👍